src/nimqml/private/dotherside.nim

```diff
@@ -351,6 +351,8 @@ proc dos_osnotification_create(): DosStatusOSNotification
 proc dos_osnotification_show_notification(vptr: DosStatusOSNotification,
   title: cstring, messsage: cstring, identifier: cstring) 
   {.cdecl, dynlib: dynLibName, importc.}
+proc dos_osnotification_show_badge_notification(vptr: DosStatusOSNotification, mentions: int) 
```

Comment:
@endulab please just update naming (on both places here) following the comment I added [here](https://github.com/status-im/dotherside/pull/64#discussion_r865610734).

Reply:
@saledjenic fixed